src/codeu/chat/DerbyStore.java

```diff
@@ -16,13 +16,16 @@
 import codeu.chat.util.Logger;
 import codeu.chat.util.Uuid;
 
+import codeu.chat.common.DerbyDatabaseInteractions;
+
 import java.io.File;
+import java.io.IOException;
 import java.sql.Connection;
 
-public class DerbyStore {
+public class DerbyStore implements DerbyDatabaseInteractions {
 	
 	private String driver = "org.apache.derby.jdbc.EmbeddedDriver";
-	private String protocol = "jdbc:derby:test";
+	private String protocol = "jdbc:derby:../test";
```

Comment:
This should probably be a parameter, that way you can use a different database for testing and for production. You also probably want a comment explaining where this is going to end up on the file system.